zeppelin-web/src/components/navbar/navbar.controller.js

```diff
@@ -115,4 +115,8 @@ angular.module('zeppelinWebApp').controller('NavCtrl', function($scope, $rootSco
   vm.loadNotes();
   $scope.checkUsername();
 
+  $rootScope.dateToString = function(date) {
```

Comment:
I thought we can have a common function for date to string which whole application can use as a standard date format.